dashboard/src/locales/messages/index.ts

```diff
@@ -192,6 +192,8 @@ export const messages = {
     'issueDetails.firstIncidentData': 'First Incident Data',
     'issueDetails.id': 'Issue Id',
     'issueDetails.issueDetails': 'Issue Details',
+    'issueDetails.issueListingInfo':
+      'The issues listed below are caused by the execution code',
```

Comment:
I'm not sure this is better but I'd like to see the word 'culprit' explicitly used here, something like

```suggestion
      'The culprit for all issues listed is code',
```